### PR TITLE
2454

### PR DESCRIPTION
**Pull request by benoit-canet**
Remove the extra properties checker from the tools so we can get rid on
the scylla.yaml to cassandra.yaml synchronisation script.

Second patch remove it.

### CODE REVIEW

File: tools/bin/cassandra.in.sh [hunk 1]
```diff
@@ -46,25 +46,12 @@ fi
...
-    # cassandra options parser go boom.
-    # Also add attributes not present in scylla.yaml
-    # but required by cassandra.
-    `dirname $0`/filter_cassandra_attributes.py \
-            "$CASSANDRA_CONF/cassandra.yaml" \
-            "$TMPCONF/scylla.yaml" \
-            > "$TMPCONF/cassandra.yaml"
-    CASSANDRA_CONF=$TMPCONF
-    fi
+cp -a $SCYLLA_CONF/scylla.yaml $SCYLLA_CONF/cassandra.yaml
```
Comment (slivne) [inline tools/bin/cassandra.in.sh]: why do we need to copy ?

**Comment by benoit-canet** [comment]
@slivne I think the tool diggest a cassandra.yaml file. Before those patches the python script was doing the "copy".

**Comment by benoit-canet** [comment]
@slivne: Note there is extra logging infos at the begining of the ouput with the default scylla configuration file

File: tools/bin/cassandra.in.sh (hunk 1 above)
Comment (slivne) [inline tools/bin/cassandra.in.sh]: @benoit-canet - so why not change that - e.g. force the code to use the scylla.yaml instead

**Comment by benoit-canet** [comment]
@slivne: iterated

**Comment by benoit-canet** [comment]
@penberg Hi pekka I am asking you to review this :)

**Comment by benoit-canet** [comment]
@penberg: can you review this please ?

**Comment by slivne** [comment]
@penberg ping

**Comment by penberg** [comment]
There are patches on the mailing lists, which I commented on.

**Pull request closed without merging**